client/landing/stepper/declarative-flow/flows/site-setup-flow/site-setup-flow.ts

```diff
@@ -746,6 +759,14 @@ const siteSetupFlow: FlowV1 = {
 			selectedGlobalStyles,
 			skippedCheckout,
 		] );
+
+		const { get, set } = useFlowState();
```

Comment:
This change is part of the useSideEffect() hook (Github way is not making it clear)